packages/orchestrator-ui-components/src/components/WfoTable/WfoBasicTable/styles.ts

```diff
@@ -48,8 +48,29 @@ export const getStyles = (theme: EuiThemeComputed) => {
             cursor: isSortable ? 'pointer' : 'not-allowed',
         });
 
+    const basicTableWithColorColumn = css(
+        {
+            '.euiTableHeaderCell': {
```

Comment:
Try to not extend on eui classes but add wfo specific ones to make the coupling with eui less. The classnames may change upstream.